packages/tocco-ui/src/Popover/Popover.js

```diff
@@ -24,13 +24,8 @@ const Popover = ({
 }) => {
   const [showToolTip, setShowToolTip] = useState(false)
 
-  const handleMouseEnter = () => {
-    setShowToolTip(true)
-  }
-
-  const handleMouseLeave = () => {
-    setShowToolTip(false)
-  }
+  const handleMouseEnter = () => setShowToolTip(true)
```

Comment:
With the new syntax you now return something. Please add curly braces to avoid this.